python/graphstorm/dataloading/dataloading.py

```diff
@@ -1378,32 +1410,36 @@ def _next_data(self, etype):
 class GSgnnNodeDataLoaderBase():
     """ The base dataloader class for node tasks.
 
-    If users want to customize the dataloader for node prediction tasks
+    If users want to customize dataloaders for their node prediction tasks,
     they should extend this base class by implementing the special methods
-    `__iter__` and `__next__`.
+    ``__iter__``, ``__next__``, and ``__len__``.
 
     Parameters
     ----------
     dataset : GSgnnData
-        The dataset for the node task.
+        The GraphStorm data for node tasks.
     target_idx : dict of Tensors
-        The target node IDs.
-    fanout : list or dict of lists
+        The target node indexes for prediction.
+    fanout : list of int, or dict of lists
```

Comment:
The doc string of fanout should be consistent between Edge and Node dataloaders.

Reply:
All were updated as `Neighbor sampling fanout. If it's a dict of list, it indicates the fanout for each edge type.`